src/components/components/audio-player/audio-player.yml

```diff
@@ -1,4 +1,4 @@
 # ball audio file by marini24 under CC 3.0
 audio_content:
-  - './ball.mp3'
-  - './ball.ogg'
+  - 'static/media/src/media/ball.mp3'
```

Comment:
@amazingrando Did you run `npm run build` before running `develop` - it will source the files to be in the `./`